lyrics/rock/november-rain.txt

```diff
@@ -49,26 +49,10 @@ When there's no one left to blame
 So never mind the darkness
 We still can find a way
 'Cause nothin' lasts forever
-Even cold November rain
+Even cold August rain

```

Comment:
@kbjarkefur, are you sure you meant to write August here? In all other cases you changed to April